udata_front/theme/gouvfr/assets/js/api/datasets.ts

```diff
@@ -52,6 +52,13 @@ export async function getOrganizationDatasets(oid: string, q: string, page: numb
   return resp.data;
 }
 
+export async function getUserDatasets(userId: string, q: string, page: number, pageSize: number, sortDirection: string) {
+  const resp = await api.get<PaginatedArray<Dataset>>(getLocalizedUrl(`datasets/`), {
+    params: { q, sort: sortDirection, page_size: pageSize, page, owner: userId }
```

Comment:
I don't think you get your private, deleted & archived datasets on this route sadly :cry: